.github/CODEOWNERS

```diff
@@ -0,0 +1,17 @@
+# GitHub CODEOWNERS. See:
+# https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners
+# Remember, the last match takes precedence.
+
+# Replace the Starcraft team with your appropriate code owners.
+
+# Global owner, used as a last resort.
+*       @canonical/starcraft
```

Comment:
Why did you choose this group instead of starcraft-reviewers? Doesn't this mean Sergio will start getting a flood of notifications again?

Reply:
Answered my own question - starcraft-reviewers can't be requested on this repo. 

Do you recall why that is?

Reply:
I think it's just a matter of it not having been added. @sergiusens what would you like to do about this?

Reply:
We could always add this repo to terraform with `starcraft-reviewers` having write access and then I can change the CODEOWNERS file.

Reply:
can we only keep the last two entries? that was my intention from docs, which has a clear owner, we shouldn't be side tracking on this now if at all possible